packages/api/src/record.ts

```diff
@@ -250,6 +250,7 @@ export class Record implements RecordModel {
   /** Record's published status (true/false) */
   get published() { return this._descriptor.published; }
 
+  /** Tags of the record */
```

Comment:
should the return type be documented? i am unsure of how we're using JSDoc and confused. are return types auto documented?

Reply:
Return type is documented, this is a getter.